<launch>

  <!-- Pass in mocap_config_file:=/path/to/config.yaml to change options. -->
  <arg name="mocap_config_file"
       default="$(find mocap_optitrack)/config/vision_pose.yaml" />

  <node pkg="mocap_optitrack"
        type="mocap_node"
        name="mocap_node"
        respawn="false"
        launch-prefix=""
        required="true">
    <rosparam file="$(arg mocap_config_file)" command="load" />
  </node>

  <node pkg="mocap_optitrack" type="correct_vis_coords"
    name="correct_quad_coords"
    args="/uncorrected_pose3 /mavros/vision_pose/pose" />

  <node pkg="mocap_optitrack" type="correct_vis_coords"
    name="correct_object_coords"
    args="/uncorrected_pose9 /object_pose" />

  <node pkg="mocap_optitrack" type="script_test"
    name="script_test" />

</launch>
